dataScripts/tessTrain/tessTrain.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+# Tested on a bare install of Ubuntu 22.04.3 LTS - 02/27/2024
+# Author: Charles@CharlesBock.com - github.com/Penguin2600
+
+# Color Stuff
+BGreen='\033[1;32m'
+NC='\033[0m'
+
+greentext () {
+   echo -e "\n${BGreen}### $1 ${NC}\n"
+}
+
+# Build tesseract
+# Upstream Docs: https://tesseract-ocr.github.io/tessdoc/Compiling-%E2%80%93-GitInstallation#installing-with-autoconf-tools
+greentext "Installing Deps and Creating File Structure"
+
+# Dont polute the directory
+mkdir -p ./tess
+cd tess
+pwd
+
+# Get Deps
+sudo apt-get install libicu-dev libpango1.0-dev libcairo2-dev
+sudo apt-get install automake ca-certificates g++ git libtool libleptonica-dev make pkg-config
+sudo apt-get install libpango1.0-dev libleptonica-dev
+
+greentext "Cloning Tesseract"
+
+git clone https://github.com/tesseract-ocr/tesseract.git 2> /dev/null || (cd tesseract ; git pull; cd ..)
+
+# Build the training tools
+# Upstream Docs: https://tesseract-ocr.github.io/tessdoc/Compiling-%E2%80%93-GitInstallation#build-with-training-tools
+greentext "Building Tesseract WITH Training Tools - This can take a long time"
+
+sudo apt-get install libicu-dev
+sudo apt-get install libpango1.0-dev
+sudo apt-get install libcairo2-dev
+
+cd tesseract
+
+./autogen.sh
+./configure
+make
+sudo make install
+sudo ldconfig
+make training
+sudo make training-install
+
+greentext "Finished Building Teseract and Training Tools"
+
+cd ..
+pwd
+
+# Install and configure tesstrain
+# Upstream Docs: https://github.com/tesseract-ocr/tesstrain?tab=readme-ov-file#choose-model-name
+
+greentext "Pulling the required ENG traineddata from github"
+wget https://github.com/tesseract-ocr/tessdata/raw/main/eng.traineddata
+sudo mv eng.traineddata /usr/local/share/tessdata
```

Comment:
Is there a way to not populate paths outside of noitool? It would be great if this would be confined to this directory. It looks like you can use `TESSDATA` variable to make it local. https://github.com/tesseract-ocr/tesstrain?tab=readme-ov-file#train

Reply:
Yes, I think that's a great idea, will incorporate.